WooCommerce/WooCommerceTests/POS/Controllers/PointOfSaleOrderControllerTests.swift

```diff
@@ -683,6 +687,7 @@ struct PointOfSaleOrderControllerTests {
 
             // We need an existing order before we can send a receipt
             await sut.syncOrder(for: .init(purchasableItems: [makeItem()]), retryHandler: { })
+            analyticsProvider.receivedEvents.removeAll()
```

Comment:
Similar to above, this test case passes after adding the following on my end:
```suggestion
            analyticsProvider.receivedEvents.removeAll()
            analyticsProvider.receivedProperties.removeAll()
```